descriptors/ModuleDescriptor-template.json

```diff
@@ -272,6 +272,11 @@
           "methods": ["GET"],
           "pathPattern": "/data-import/uploadUrl",
           "permissionsRequired": ["data-import.uploadUrl.get"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/data-import/uploadUrl/continue",
```

Comment:
Very Minor comment --- maybe this could be changed from
`/data-import/uploadUrl/continue` to
`/data-import/uploadUrl/part`